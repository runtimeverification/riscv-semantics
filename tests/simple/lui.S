#include "simple.h"

START_TEXT
        lui x0, 0x10
        lui x1, 0x80000
        lui x2, 0x70000
END_TEXT
