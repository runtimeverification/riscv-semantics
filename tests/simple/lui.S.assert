regs: {0: 0x0, 1: 0x80000000, 2: 0x70000000}
