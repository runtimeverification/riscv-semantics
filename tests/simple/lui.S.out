<generatedTop>
  <riscv>
    <instrs>
      .K
    </instrs>
    <regs>
      1 |-> W ( 2147483648 ) 2 |-> W ( 1879048192 )
    </regs>
    <pc>
      W ( 65632 )
    </pc>
    <mem>
      W ( 65536 ) |-> 127 W ( 65537 ) |-> 69 W ( 65538 ) |-> 76 W ( 65539 ) |-> 70 W ( 65540 ) |-> 1 W ( 65541 ) |-> 1 W ( 65542 ) |-> 1 W ( 65543 ) |-> 0 W ( 65544 ) |-> 0 W ( 65545 ) |-> 0 W ( 65546 ) |-> 0 W ( 65547 ) |-> 0 W ( 65548 ) |-> 0 W ( 65549 ) |-> 0 W ( 65550 ) |-> 0 W ( 65551 ) |-> 0 W ( 65552 ) |-> 2 W ( 65553 ) |-> 0 W ( 65554 ) |-> 243 W ( 65555 ) |-> 0 W ( 65556 ) |-> 1 W ( 65557 ) |-> 0 W ( 65558 ) |-> 0 W ( 65559 ) |-> 0 W ( 65560 ) |-> 84 W ( 65561 ) |-> 0 W ( 65562 ) |-> 1 W ( 65563 ) |-> 0 W ( 65564 ) |-> 52 W ( 65565 ) |-> 0 W ( 65566 ) |-> 0 W ( 65567 ) |-> 0 W ( 65568 ) |-> 200 W ( 65569 ) |-> 1 W ( 65570 ) |-> 0 W ( 65571 ) |-> 0 W ( 65572 ) |-> 8 W ( 65573 ) |-> 0 W ( 65574 ) |-> 0 W ( 65575 ) |-> 0 W ( 65576 ) |-> 52 W ( 65577 ) |-> 0 W ( 65578 ) |-> 32 W ( 65579 ) |-> 0 W ( 65580 ) |-> 1 W ( 65581 ) |-> 0 W ( 65582 ) |-> 40 W ( 65583 ) |-> 0 W ( 65584 ) |-> 5 W ( 65585 ) |-> 0 W ( 65586 ) |-> 4 W ( 65587 ) |-> 0 W ( 65588 ) |-> 1 W ( 65589 ) |-> 0 W ( 65590 ) |-> 0 W ( 65591 ) |-> 0 W ( 65592 ) |-> 0 W ( 65593 ) |-> 0 W ( 65594 ) |-> 0 W ( 65595 ) |-> 0 W ( 65596 ) |-> 0 W ( 65597 ) |-> 0 W ( 65598 ) |-> 1 W ( 65599 ) |-> 0 W ( 65600 ) |-> 0 W ( 65601 ) |-> 0 W ( 65602 ) |-> 1 W ( 65603 ) |-> 0 W ( 65604 ) |-> 100 W ( 65605 ) |-> 0 W ( 65606 ) |-> 0 W ( 65607 ) |-> 0 W ( 65608 ) |-> 100 W ( 65609 ) |-> 0 W ( 65610 ) |-> 0 W ( 65611 ) |-> 0 W ( 65612 ) |-> 5 W ( 65613 ) |-> 0 W ( 65614 ) |-> 0 W ( 65615 ) |-> 0 W ( 65616 ) |-> 0 W ( 65617 ) |-> 16 W ( 65618 ) |-> 0 W ( 65619 ) |-> 0 W ( 65620 ) |-> 55 W ( 65621 ) |-> 0 W ( 65622 ) |-> 1 W ( 65623 ) |-> 0 W ( 65624 ) |-> 183 W ( 65625 ) |-> 0 W ( 65626 ) |-> 0 W ( 65627 ) |-> 128 W ( 65628 ) |-> 55 W ( 65629 ) |-> 1 W ( 65630 ) |-> 0 W ( 65631 ) |-> 112 W ( 65632 ) |-> 19 W ( 65633 ) |-> 0 W ( 65634 ) |-> 0 W ( 65635 ) |-> 0
    </mem>
  </riscv>
  <test>
    <haltCond>
      ADDRESS ( W ( 65632 ) )
    </haltCond>
    <halt>
      true
    </halt>
  </test>
  <generatedCounter>
    0
  </generatedCounter>
</generatedTop>